src/MacroTools/Extensions/RectangleExtensions.cs

```diff
@@ -43,5 +43,22 @@ public static List<unit> PrepareUnitsForRescue(this Rectangle area, player ownin
       }
       return group;
     }
+    
+    /// <summary>
+    /// Sets the owner of all units inside the specified <paramref name="area"/> to the specifed <see cref="player" />
+    /// </summary>
+    /// <param name="area"></param> The area the units are in
+    /// <param name="whichPlayer"></param> The player to gain control of the units
+    public static void RescueNeutralUnitsInRect(this Rectangle area, player whichPlayer)
```

Comment:
I've been thinking about this, and I think we should remove it and adopt the pattern @TracyTheBoy introduced for every Quest of this nature. Check out this ticket: https://github.com/AzerothWarsLR/WarcraftLegacies/issues/144

Basically, whenever a Quest with a rescue area is constructed, it should grab all of the units in that area and put them into a group. Then when the Quest is completed, it should rescue that group instead of rescuing units in the area. This avoids the possibility of us accidentally rescuing units that wander into the area later on.